WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -160,6 +169,11 @@ public void onClick(View v) {
             }
         });
 
+        if (mSelfHosted) {
```

Comment:
This duplicates code that occurs in the `mAddSelfHostedButton` click handler [above](https://github.com/wordpress-mobile/WordPress-Android/pull/3735/files#diff-d1955c04c369a8c269a4d81a5fdbe99fR160). Seems like a good candidate for a separate `setIsSelfHosted` method.